wolftpm/tpm2_wrap.h

```diff
@@ -2348,6 +2348,28 @@ WOLFTPM_API int wolfTPM2_ClearCryptoDevCb(WOLFTPM2_DEV* dev, int devId);
 
 #endif /* WOLF_CRYPTO_CB */
 
+#ifndef WOLFTPM2_NO_HEAP
+WOLFTPM_API WOLFTPM2_DEV *wolfTPM2_New(void);
+WOLFTPM_API int wolfTPM2_Free(WOLFTPM2_DEV *dev);
+WOLFTPM_API WOLFTPM2_KEYBLOB* wolfTPM2_GetNewKeyBlob(void);
```

Comment:
For the API name's I would prefer `wolfTPM2_NewKeyBlob` and `wolfTPM2_FreeKeyBlob`. Same pattern for others below.